cmd/ethrex/initializers.rs

```diff
@@ -237,7 +237,7 @@ pub fn get_bootnodes(opts: &Options, network: &Network, data_dir: &str) -> Vec<N
 
     match read_node_config_file(config_file) {
         Ok(ref mut config) => bootnodes.append(&mut config.known_peers),
-        Err(e) => warn!("Could not read from peers file: {e}"),
+        Err(e) => debug!("Could not read from peers file: {e}"),
```

Comment:
we do want to log an error if there is an actual error. But we want to handle the non existance of a file separately (it is not an error, it's a valid usecase)